docs/src/main/paradox/java/cluster-http-management.md

```diff
@@ -0,0 +1,179 @@
+<a id="http-cluster-management"></a>
```

Comment:
For this I don't think we need separate java and scala pages. Instead we can use Java / Scala tabs in Paradox code includes. See example: https://raw.githubusercontent.com/johanandren/alpakka/61aeb1e06c095759447c2b20dfeb03cf76a3f9ca/docs/src/main/paradox/amqp.md